tests/ReactTestApp/ios/ReactTestAppTests/ReactTestAppTests.m

```diff
@@ -123,8 +88,11 @@ + (XCTestSuite *)defaultTestSuite {
 
     @autoreleasepool {
         static NSDictionary *testCaseNames;
+        __weak AppDelegate *delegate = (AppDelegate*)[[UIApplication sharedApplication] delegate];
```

Comment:
None of the local variables here should be __weak.

Reply:
Ok, good to know.  I'll update those.